codalab/server/bundle_manager.py

```diff
@@ -619,10 +624,10 @@ def _schedule_run_bundles(self):
         self._restage_stuck_starting_bundles(workers)
         self._bring_offline_stuck_running_bundles(workers)
         self._acknowledge_recently_finished_bundles(workers)
-        staged_bundles_to_run = self._get_staged_bundles_to_run()
+        staged_bundles_to_run, user_info_cache = self._get_staged_bundles_to_run()
```

Comment:
To make the function signatures more uniform and clean, I'd create `user_info_cache = {}` and pass it in to all the functions that need it; it's just a variable that will get accessed and updated as needed.